claim-check/code-samples/sample-3/readme.md

```diff
@@ -1,108 +1,107 @@
+Make sure you have WSL (Windows System For Linux) installed and have AZ CLI version > 2.0.50. Before running any script also make sure you are authenticated on AZ CLI using
 
-```bash
-az login
-```
+### Steps
 
-and have selected the Azure Subscription you want to use for the tests:
+1. Clone this repository to your workstation and navigate to the working directory.
 
-```bash
-az account list --output table
-az account set --subscription "<YOUR SUBSCRIPTION NAME>"
-```
+   ```shell
+   git clone https://github.com/mspnp/cloud-design-patterns
+   cd cloud-design-patterns/claim-check/code-samples/sample-3
+   ```
 
-## Clone the sample project
+1. Log into Azure and create an empty resource group.
 
-Clone the repository and open the code-samples directory from your command line tool.
+   ```azurecli
+   az login
+   az account set -s <Name or ID of subscription>
 
-```bash
-git clone https://github.com/mspnp/cloud-design-patterns.git
-cd claim-check/code-samples/sample-3
-```
+   NAME_PREFIX=<unique value between three to five characters>
+   az group create -n "rg-${NAME_PREFIX}" -l eastus2
 
-## Run Azure Setup Script
+  ```
 
-Run the azure setup script to get the resources deployed and everything set up
+1. Deploy the supporting Azure resources.
 
-```bash
-./sample-3-azure-setup.sh <unique-name>
-```
-
-_unique-name_ should be something that is unlikely to be used by someone else. This is needed to make sure that no conflict with other people running the same sample at the same time will arise. The name should only contains numbers and letters should not be longer than 12 characters (as additional text will be added by the script itself to identify each created resource) If you're not sure about what to use here, you can just generate a random string using the following bash command:
+   ```azurecli
+   CURRENT_USER_OBJECT_ID=$(az ad signed-in-user show -o tsv --query id)
 
-```bash
-echo `openssl rand 5 -base64 | cut -c1-7 | tr '[:upper:]' '[:lower:]' | tr -cd '[[:alnum:]]._-'`
-```
+   # This could take a few minutes
+   az deployment group create -n deploy-claim-check -f bicep/main.bicep -g "rg-${NAME_PREFIX}" -p namePrefix=$NAME_PREFIX principalId=$CURRENT_USER_OBJECT_ID
+   ```
 
-This script will create
+1. Configure the samples to use the created Azure resources.
 
-* a resource group
-* a V2 storage account
-* a storage account container
-* a service bus namespace, service bus and service bus queue
-* a function app in an app service plan
-* an application insights service
+   ```shell
+   sed "s/{EVENT_HUBS_NAMESPACE}/evhns-${NAME_PREFIX}/g" ClientConsumer2/appsettings.json.template >ClientConsumer2/appsettings.json
+   sed -i "s/{EVENT_PROCESSOR_STORAGE_ACCOUNT_NAME}/st${NAME_PREFIX}ehub/g" ClientConsumer2/appsettings.json
 
-## Running the sample
+   sed "s/{EVENT_HUBS_NAMESPACE}/evhns-${NAME_PREFIX}/g" FunctionConsumer2/local.settings.json.template > FunctionConsumer2/local.settings.json
```

Comment:
There is no Event Hub in this one. Was this supposed to be where you set the service bus details?

Also referencing "2" instead of "3" here.